Makefile

```diff
@@ -1000,6 +1000,10 @@ RELEASE_TAG ?= $(shell git describe --abbrev=0 2>/dev/null)
 ifneq (,$(findstring -,$(RELEASE_TAG)))
     PRE_RELEASE=true
 endif
+## Check if previous tag is provided, if not set empty
+ifeq (,$(findstring -,$(PREVIOUS_VERSION_TAG)))
+    PREVIOUS_VERSION_TAG=""
```

Comment:
`PREVIOUS_TAG` is not used anywhere, I believe it can be removed, and `PREVIOUS_VERSION_TAG` is tag which is used to generate release notes for RC/BETA release, for example `RELEASE_TAG=v1.7.0-rc.1 PREVIOUS_VERSION_TAG=tags/v1.7.0-rc.0 make release-notes`

Reply:
Yup PREVIOUS_TAG was used in some earlier iteration of the release notes tool